app/workers/my_anime_list_list_worker.rb

```diff
@@ -0,0 +1,16 @@
+class MyAnimeListListWorker
+  include Sidekiq::Worker
+
+  def perform(user_id)
```

Comment:
hey.. if you read like 9 lines down in the private method IT IS RIGHT THERE?!?!! 🖕